cmd/cdi-importer/importer.go

```diff
@@ -29,13 +35,30 @@ func init() {
 func main() {
 	defer glog.Flush()
 
+	certsDirectory, err := ioutil.TempDir("", "certsdir")
+	if err != nil {
+		panic(err)
+	}
+	defer os.RemoveAll(certsDirectory)
+
+	keyFile, certFile, err := keys.GenerateSelfSignedCert(certsDirectory, "importer", "pod")
```

Comment:
Basically I followed what kubevirt did in this instance, tell prometheus to ignore the signing. Since there is no sensitive data at all, and we don't have any infra structure to obtain a proper certificate at this point, they went (and thus I went) with a self signed cert.